src/crawlers/browser_crawler.js

```diff
@@ -285,7 +286,7 @@ class BrowserCrawler extends BasicCrawler {
         this.postNavigationHooks = postNavigationHooks;
 
         if (useSessionPool) {
-            this.persistCookiesPerSession = persistCookiesPerSession || true;
+            this.persistCookiesPerSession = persistCookiesPerSession !== undefined ? persistCookiesPerSession : true;
```

Comment:
When `persistCookiesPerSession` is `undefined`, this will end up

```js
this.persistCookiesPerSession = undefined
```

Reply:
I am not quite following... If `persistCookiesPerSession` is `undefined` -> `this.persistCookiesPerSession` will be set to `true` when session pool is used, and will be set to `false` if `useSessionPool: false`.

Regarding the defaults - it actually becomes less readable - if I set the default to true (like I did before) - I have to perform the check with `options.persistCookiesPerSession` and then re-assign the value. If it's `false` by default - I would also have to re-assign the value for certain scenarios. Now it's always true or false in the end depending on actual usage of session pool (without the default value).

Reply:
Yup, sorry. Read it as `===`